.gitignore

```diff
@@ -7,16 +7,12 @@ vpp-manager/images/*/vpp-manager
 vpp-manager/images/*/vppdev.sh
 vpp-manager/vpp_build/
 *.deb
-vpplink/binapi/bin/
 multinet-monitor/watcher
 
 test/k8s-e2e-tests/kubernetes-*
 test/vagrant/local-config.env
 test/vagrant/.vagrant
 
-# cherry-pick patch caching
-vpplink/binapi/.cherries-cache
```

Comment:
I think this is still produced by `vpplink/generated/vpp_clone_current.sh` (and should still be gitignored)